doc/manual/manual.tex

```diff
@@ -298,16 +298,19 @@ \subsection{Referencing \aspect{}}
 In addition, many have contributed to the development of \aspect{} and deserve credit
 for their work.
 To this end, we ask that if you publish results that were obtained
-to some part using \aspect{}, cite the following, canonical reference for
+to some part using \aspect{}, cite the following, canonical reference (\cite{heister_aspect_methods2}) for
 this software:
 \begin{lstlisting}[frame=single,language=tex]
-@Article{KHB12,
-  author =       {M. Kronbichler and T. Heister and W. Bangerth},
-  title =        {High Accuracy Mantle Convection Simulation through Modern Numerical Methods},
-  journal =      {Geophysics Journal International},
-  year =         2012,
-  volume =       191,
-  pages =        {12--29}
+@Article{heister_aspect_methods2,
+  Title                    = {{High Accuracy Mantle Convection Simulation through Modern Numerical Methods. II: Realistic Models and Problems}},
```

Comment:
Correct. That's what the typical styles all suggest to happen for article titles. (They differ on how to treat book titles.) That's why bibtex does it automatically.